editor/src/clj/editor/console.clj

```diff
@@ -93,6 +94,51 @@
   []
   (swap! pending-atom assoc :clear true :entries [] :index 0))
 
+(def ^:private remote-log-pump-thread (atom nil))
+(def ^:private console-stream (atom nil))
+
+(defn current-stream? [stream] (= @console-stream stream))
```

Comment:
this code moved from `app-veiw` except `current-stream?` function. This is a new one